database/schema.sql

```diff
@@ -49,7 +49,7 @@ CREATE TABLE dbo.hbmember (
 	forename varchar(32) NULL,
 	find_name varchar(32) NULL,
 	type smallint NULL,
-	birth_date timestamp NULL,
+	birth_date varchar(37) NULL,
```

Comment:
Great spot. Was this from DMS. 
I would have assumed this to be a timestamp 😄

Reply:
Yeah me too!